cypher/remove.md

```
@@ -0,0 +1,68 @@
+
+Similar to removing a single label from a node we can use the REMOVE clause
+to remove multiple labels in one go
+
+```cypher
+MATCH (n :Player {name: 'Lori'})
+REMOVE n:Admin:Player
+RETURN n.name, labels(n)
+```
+
+Result:
+|n.name|abels(n)|
```

Markdownlint (0.35.0)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)